pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -161,15 +161,7 @@ let
     patches = [
       ./patches/no-build-timestamps.patch # Optional patch to use SOURCE_DATE_EPOCH in compute_build_timestamp.py (should be upstreamed)
       ./patches/widevine-79.patch # For bundling Widevine (DRM), might be replaceable via bundle_widevine_cdm=true in gnFlags
-      # ++ optional (versionRange "68" "72") (githubPatch "<patch>" "0000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
Worth keeping this comment to describe how to use githubPatch?

Reply:
I thought about it but it's not that useful IOM . I might re-add it in the future with a more useful formatting/structure.

Reply:
SG 👍